src/lib/holocene/card.svelte

```diff
@@ -1,9 +1,7 @@
-<div class="card surface-primary {$$props.class}">
-  <slot />
-</div>
+<script lang="ts">
+  let { class: className = '', children } = $props();
+</script>
 
-<style lang="postcss">
-  .card {
-    @apply border border-subtle p-4;
-  }
-</style>
+<div class="surface-primary border border-subtle p-4 {className}">
```

Comment:
Sure className is probably the more acceptable variable name. But 'klass' gave me so much Ruby nostalgia. Ahh well, className is the right move.

Reply:
The Rosses are at war on this one

Reply:
If there is a war I concede. "Baby Face Ross" for sure has the more correct opinion in this matter.